LifeUtils/LifeUtils/Minecraft/MinecraftVersion.cs

```diff
@@ -19,7 +19,7 @@ public static class MinecraftVersion
         ///     Note: Changes in every update.
         /// </summary>
         public static readonly Version Latest =
-            new Version(1, 13, 2);
+            new Version(1, 14);
```

Comment:
Spigot 1.14 is not released yet, and we should wait for 1.14.2 or such. But this can be merged early. No early release though. The 1.0.1 release will include this (and 1.14.2) when released.